tests/Properties/ByteString.hs

```diff
@@ -81,12 +82,16 @@ tests =
 #ifndef BYTESTRING_CHAR8
   , testProperty "toFilePath >>= fromFilePath" $
     \x -> ioProperty $ do
-      r <- B.toFilePath x >>= B.fromFilePath
-      pure (r === x)
+      enc <- getFileSystemEncoding
+      if textEncodingName enc == "ASCII" then pure (property True) else do
```

Comment:
Is there a more explicit/readable way to skip this test than `property True`? A comment explaining _why_ the test is skipped would be helpful too.